models/issue_comment.go

```diff
@@ -724,7 +724,7 @@ func (c *Comment) CodeCommentURL() string {
 }
 
 // LoadPushCommits Load push commits
-func (c *Comment) LoadPushCommits() (err error) {
+func (c *Comment) LoadPushCommits(ctx context.Context) (err error) {
```

Comment:
This method should be moved to `service` package.

Reply:
The purpose of this very large PR is to refactor only to begin passing context down the callstack - I think that's enough changes without adding in other major refactors.